RAGchain/benchmark/dataset/msmarco.py

```diff
@@ -60,7 +60,6 @@ def __init__(self, run_pipeline: BaseRunPipeline,
                            + self.retrieval_gt_ragas_metrics + self.retrieval_no_gt_ragas_metrics
                            + self.retrieval_gt_metrics_rank_aware + self.answer_no_gt_metrics +
```

Comment:
answer_no_gt_metrics is 'ragas' metrics. So it needs to be support_metrics, but not default_metrics.
Plus, you are missing answer_gt_metrics like BLEU for support metrics.
You have ground truth answer, so you can support those metrics.